modules/examples/eureka/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.black]
```

Comment:
as per the latest standard, we have retired isort and black and moved to `ruff`. Can you please check any of the existing modules in ADDF repo and replace here?

Reply:
Yes, updated to `[tool.ruff]`